### PR TITLE
Deprecate PIF repo

### PR DESCRIPTION
**Pull request by rogeruiz**
This patch adds the deprecation message to this repository and points
users to the `18F/vote-2016` repository. This will most likely be the last
commit on the `presidential-innovation-fellows/vote-2016` repository as
the 18F fork will contain all new work going forward.

### CODE REVIEW

**Comment by rogeruiz** [comment]
The _work in progress_ title will stay on this PR until we have a final strategy around deployment. :+1:

File: README.md [hunk 1]
```diff
@@ -1,11 +1,24 @@
-#Vote 2016
+# Vote 2016
 
-To get the SASS working, run `sass --watch assets/stylesheets/sass:assets/stylesheets/output` from the project directory.
+**This repository has been deprecated. Please use the [`18F/vote-2016`] [18f-fork]
+fork to follow development for this project or to file any issues. You must open
```
Comment (kategarklavs) [inline README.md]: I might change "You must open all PRs" with "Please open pull requests against the master ..." (to sound a bit friendlier and more conversational).

File: README.md [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+```
 
 ### Public domain
 
 This project is in the worldwide [public domain](LICENSE.md). As stated in [CONTRIBUTING](CONTRIBUTING.md):
 
-> This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
->
-> All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
+> This project is in the public domain within the United States, and copyright
```
Comment (kategarklavs) [inline README.md]: Hey! I might consider breaking this into two sentences:

"This project is in the public domain within the United States. Worldwide, copyright and related rights are waived through the CC0 1.0 ... dedication."

(The current placement of _worldwide_ is throwing me a bit.)

**Comment by kategarklavs** [comment]
@rogeruiz Just took a look at this PR and made a few minor recommendations. No additional comments from me!

**Comment by rogeruiz** [comment]
~~okay @stroupaloop This is ready to be merged in!~~ :+1:

**Comment by rogeruiz** [comment]
Updated with your suggestions. Thanks @kategarklavs !

**Comment by kategarklavs** [comment]
You're most welcome!

**Comment by rogeruiz** [comment]
Closing this PR for now.

**Pull request closed without merging**